extension-style/src/main/java/com/mapbox/maps/extension/style/image/ImageExtensionImpl.kt

```diff
@@ -98,6 +98,24 @@ class ImageExtensionImpl(private val builder: Builder) : StyleContract.StyleImag
       this.internalImage = Image(bitmap.width, bitmap.height, byteBuffer.array())
     }
 
+    /**
+     * Set 9-patch bitmap.
+     *
+     * This will automatically set correct [internalImage], [content], [stretchX] and [stretchY]
+     * with no need to calculate them manually.
+     */
+    fun bitmapNinePatch(bitmap: Bitmap) = apply {
```

Comment:
What if user will bitmapNinePatch but then change stretches settings? 
Maybe 9patch should be separate imageExtension without a possibility to override those?

Reply:
imho no
but guess it's discussable, I can't say I have very strong opinion, I'm OK with both options.
@mapbox/maps-android wdyt?

Reply:
Will be creating separate extension function, it indeed looks more logical.
Thanks @Chaoba and @yunikkk for pointing it out!